### PR TITLE
Prettier code and added code to dontOpenMe.html and updated Discord.js

### PR DESCRIPTION
**Pull request by yuto0214w**
https://discord.com/channels/671494961577066498/712740830552522838/779074983527710721

#### Prettiered .js .json .yml .css .html
.js .yml is single-quote `prettier --write "**/*.{js,yml}" --single-quote --tab-width=4`
.json .css .html is double-quote `prettier --write "**/*.{json,css,html}" --tab-width=4`

#### Tell the user to open dontOpenMe.html "This isn't going to work...."
Added
```html
    <script type="text/javascript">
      if (typeof process === 'undefined') { // Judging if user accessed from Electron
        alert("This isn't going to work....");
      }
    </script>
```

#### Updated Discord.js
Updated Discord.js 12.2.0 to Discord.js 12.4.1

### CODE REVIEW

File: js/rcMenuFuncs.js [hunk 1]
```diff
@@ -16,131 +16,134 @@
...
-    pin
-      ? selectedChan.messages.cache.get(id).pin()
-      : selectedChan.messages.cache.get(id).unpin();
-  } catch (e) {
-    console.log(e);
-  }
+    try {
+        pin
+            ? selectedChan.messages.cache.get(id).pin()
+            : selectedChan.messages.cache.get(id).unpin();
```
Comment (SebOuellette) [inline js/rcMenuFuncs.js:101]: What the f--- kinda ternary operation did daniel add here...?? May as well just use a normal if statement at this point.

**Review by SebOuellette — COMMENTED** [review]
Don't worry about changing the if statement or ternary, That's not your fault so I'll do it later.

File: js/buildTeamMemberCards.js [hunk 2]
```diff
@@ -13,129 +13,129 @@
...
 
     // Event listeners
-    once.addEventListener('click', () => {
-      bot.owner = m.user;
-      funky();
+    oneTimeButton.addEventListener('click', async () => {
+        let token = input.value;
+        let error = [false, 'none'];
+
+        if (global.bot === undefined) error = await load(token);
```
[the remark this answers is not in the document]
Comment (LukasDoesDev) [reply js/buildTeamMemberCards.js:121]: If it doesn't use `{` or `}` (Don't remember the name), it usually stays on the same line.

[the remark this answers is not in the document]
Comment (SebOuellette) [reply js/buildTeamMemberCards.js:121]: Cool, I don't like the way that looks so I don't want it like that.

[the remark this answers is not in the document]
Comment (LukasDoesDev) [reply js/buildTeamMemberCards.js:121]: ok

**Comment by LukasDoesDev** [comment]
668 lines added. Haha

**Comment by yuto0214w** [comment]
I don't recall why I closed the pull request...
I may have clicked by mistake

**Comment by SebOuellette** [comment]
Great, looks good! Thanks for the help @yuto0214w !

**Comment by SebOuellette** [comment]
_Didn't mean to close_

File: js/rcMenuFuncs.js (hunk 1 above)
Comment (Splatterxl) [reply js/rcMenuFuncs.js:101]: That checks if `pin` exists, then if it does, pins the message, otherwise, does not.

**Pull request merged**